partners_api/facebook_ads.hpp

```diff
@@ -12,6 +12,13 @@ class TypesHolder;
 
 namespace facebook
 {
+struct TypeAndlevel
+{
+  TypeAndlevel(uint32_t type, uint32_t level) : m_type(type), m_level(level) {}
+  uint32_t m_type = 0;
```

Comment:
Blank line between ctor and fields. BTW, do we need default initialization, in absence of default ctor?